File: server/freeScoreBoard7/Core/DB/TEreignisse.cs
namespace FreeScoreBoard.Core.DB
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("TEreignisse")]
    public partial class TEreignisse
    {
        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        public long ID { get; set; }

        public int TurnierNr { get; set; }

        public int Spiel { get; set; }

        [StringLength(2147483647)]
        public string Mannschaft { get; set; }

        [StringLength(2147483647)]
        public string Spielzeit { get; set; }

        [StringLength(2147483647)]
        public string Spielzeitrichtung { get; set; }

        public DateTime CPUZeit { get; set; }

        [StringLength(2147483647)]
        public string Ereignistyp { get; set; }

        [StringLength(2147483647)]
        public string Spieler { get; set; }

        [StringLength(2147483647)]
        public string Details { get; set; }

        [StringLength(2147483647)]
        public string Spielabschnitt { get; set; }

        public int TurnierID { get; set; }
    }
}

File: control page/src/turniere/dataClasses/TEreignisse.js
export default class TEreignisse {
  constructor(
    _ID,
    _TurnierNr,
    _Spiel,
    _Mannschaft,
    _Spielzeit,
    _Spielzeitrichtung,
    _CPUZeit,
    _Ereignistyp,
    _Spieler,
    _Details,
    _Spielabschnitt = 1,
    _TurnierID,
  ) {
    this.ID = _ID;
    this.TurnierNr = _TurnierNr;
    this.Spiel = _Spiel;
    this.Mannschaft = _Mannschaft;
    this.Spielzeit = _Spielzeit;
    this.Spielzeitrichtung = _Spielzeitrichtung;
    this.CPUZeit = _CPUZeit;
    this.Ereignistyp = _Ereignistyp;
    this.Spieler = _Spieler;
    this.Details = _Details;
    this.Spielabschnitt = _Spielabschnitt;
    this.TurnierID = _TurnierID;
  }
}
